src/VideoNativeComponent.ts

```diff
@@ -3,13 +3,45 @@ import type {
   NativeSyntheticEvent,
   ViewProps,
 } from 'react-native';
+import type {
+  Drm,
+  DebugConfig,
+  Headers,
+  TextTracks,
+  SelectedTextTrack,
+  SelectedVideoTrack,
+  SubtitleStyle,
+  BufferConfig,
+  SelectedAudioTrack,
+} from './types/video';
 import {NativeModules, requireNativeComponent} from 'react-native';
+import type {
+  OnAudioFocusChangedData,
+  OnAudioTracksData,
+  OnBandwidthUpdateData,
+  OnVideoBufferData,
+  OnExternalPlaybackChangeData,
+  OnGetLicenseData,
+  OnLoadData,
+  OnLoadStartData,
+  OnPictureInPictureStatusChangedData,
+  OnPlaybackData,
+  OnPlaybackStateChangedData,
+  OnProgressData,
+  OnReceiveAdEventData,
+  OnSeekData,
+  OnTextTracksData,
+  OnTimedMetadataData,
+  OnVideoAspectRatioData,
+  OnVideoErrorData,
+  OnVideoTracksData,
+} from './types/events';
+import type ResizeMode from './types/ResizeMode';
+import type FilterType from 'react-native-video/src/types/FilterType';
```

Comment:
should be relative path
```suggestion
import type FilterType from './types/FilterType';
```